kafka-rest/src/test/java/io/confluent/kafkarest/integration/AuthorizationErrorTest.java

```diff
@@ -72,7 +71,14 @@
   @Before
   public void setUp() throws Exception {
     super.setUp();
-    createTopic(TOPIC_NAME, 1, (short) 1);
+    Properties properties = restConfig.getAdminProperties();
+    properties.put(AdminClientConfig.BOOTSTRAP_SERVERS_CONFIG, brokerList);
+    properties.put(
+        "sasl.jaas.config",
+        "org.apache.kafka.common.security.plain.PlainLoginModule required "
+            + " username=\"admin\""
+            + " password=\"admin-secret\";");
```

Comment:
Can we extract this to a constant? It's used in other places in this file too.

Reply:
There's also the `overrideKafkaRestConfigs`

Reply:
Done.